src/test/java/apoc/load/LoadGoogleCloudStorageTest.java

```diff
@@ -0,0 +1,70 @@
+package apoc.load;
+
+import apoc.util.TestUtil;
+import org.junit.*;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.test.TestGraphDatabaseFactory;
+
+import static apoc.load.LoadCsvTest.assertRow;
+import static apoc.util.MapUtil.map;
+import static apoc.util.TestUtil.testResult;
+import static org.junit.Assert.assertFalse;
+
+public class LoadGoogleCloudStorageTest {
+
+    private GraphDatabaseService db;
+
+    private static final String BUCKET_NAME = System.getenv("APOC_GC_BUCKET_NAME");
+
+    @BeforeClass
+    public static void before() {
+        Assume.assumeNotNull(BUCKET_NAME);
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        db = new TestGraphDatabaseFactory().newImpermanentDatabaseBuilder()
```

Comment:
do we need file import enabled for remote loads, I'm not sure how we do it with other options?

Reply:
It's not necessary, I'll remove it